src/config.ts

```diff
@@ -35,10 +35,10 @@ class EnvVars {
 	GOOGLE_CLIENT_SECRET: string;
 
 	@IsString()
-	JWT_SECRET: string;
+	DATABASE_URL: string;
```

Comment:
Database url was not changed, move the paseto secret up so the history is more clear